### ISSUE TITLE
State counters in ExperimentManager

### ISSUE THREAD

**Issue by MickyHoltzman** [Anna4142/PrisonerDilemmaPy#35]
Currently we are counting how many time the mouse visited decision and center states. As far as I can see we are not doing anything with these counters. I think they are not needed. Should we delete them?

**Comment by Anna4142** [comment]
yes seems unnecessary .was something carried forward from the previous code so we can delete it

**Comment by MickyHoltzman** [comment]
I deleted it. We can close this issue.
